### PR TITLE
Override config values with bootstrap object values.

### PR DESCRIPTION
**Pull request by curioussavage**
:eyeglasses:  @ajacksified

### CODE REVIEW

File: assets/js/client.es6.js [hunk 1]
```diff
@@ -55,6 +56,12 @@ function initialize(bindLinks) {
 
     config.mountPoint = document.getElementById('app-container');
 
+    _.forOwn(config, function(val, key) {
+      if (bootstrap[key]) {
+        config[key] = bootstrap[key]
+      }
+    })
```
Comment (ajacksified) [inline assets/js/client.es6.js]: semicolon here and above

**Comment by ajacksified** [comment]
:nail_care:

**Comment by curioussavage** [comment]
:haircut:  

Was in a little too much of a hurry to push that I guess.

File: assets/js/client.es6.js [hunk 2]
```diff
@@ -55,6 +56,12 @@ function initialize(bindLinks) {
 
     config.mountPoint = document.getElementById('app-container');
 
+    _.forOwn(config, function(val, key) {
+      if (bootstrap[key]) {
+        config[key] = bootstrap[key]
```
Comment (ajacksified) [inline assets/js/client.es6.js]: semicolon here, not after the if statement

**Comment by ajacksified** [comment]
:nail_care:

**Comment by curioussavage** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish:

**Comment by ajacksified** [comment]
Merged: https://github.com/reddit/reddit-mobile/commit/bd71dc135f3976a104fb57d7bc4bf075a61cc6ab

**Pull request closed without merging**